storage/fs.go

```diff
@@ -55,11 +58,19 @@ func newFileSystemManager(database database) databaseFileSystemManager {
 	fm := newFlushManager(database)
 	cm := newCleanupManager(database, fm)
 
+	var jitter time.Duration
+	if maxJitter := opts.FileOpOptions().Jitter(); maxJitter > 0 {
```

Comment:
Nit: Might be worth pulling in the epoch based rand - would be much cheaper than creating a new rand object each time.